Signal/src/UserInterface/Strings.swift

```diff
@@ -15,16 +15,13 @@ import Foundation
 
```

Comment:
Can we rename this class to `Strings` or rename the source file to `CallStrings.swift`?